### ISSUE TITLE
add loader overlay for views

### ISSUE THREAD

**Issue by mattkime** [meetup/sassquatch2#250]
```
.view--loading {
`   &:before {
`   `   content: " ";
`   `   position: absolute;
`   `   top:50%;
`   `   left:50%;
`   `   //background: $C_blue;
`   `   //border-radius: 100%;
`   `   //height: $space-half;
`   `   //width: $space-half;
`   `   z-index: $Z_top - 1;
`   `   //animation: dotAnim 500ms ease -125ms infinite alternate;
`   `   width: 36px;
`   `   height: 36px;
`   `   border-radius: 999px;
`   `   box-sizing: border-box;
`   `   border: solid 3px rgba(0, 0, 0, 0.2);
`   `   border-top-color: black;
animation: spin-circle .5s infinite linear;
margin: 0 auto;
`   }
`   &:after {
`   `   content: " ";
`   `   display: block;
`   `   position: absolute;
`   `   left: 0;
`   `   right: 0;
`   `   top: 0;
`   `   bottom: 0;
`   `   background-color: rgba(255,255,255,0.7);
`   `   z-index: $Z_top - 2;
`   }

}
```

this is mostly done but i'm going on vacation for a week. leaving here so a generous soul may continue the work.

**Comment by akdetrick** [comment]
fixed in `develop`

**Comment by jmhammons** [comment]
Props for making this. I need something like this for `/register` so its addition is very timely.

Couple questions / observations after trying to use this in my branch.
- It doesn't seem perfectly centered horizontally (most noticeable on small screens)
- On a `.view` / page that requires scrolling, the loader seems to vertically center itself within the entire view / page height. And the result is the spinner is often not visible. Maybe I'm not using this correctly or I need to do something else? Any advice would def be appreciated.

**Comment by mattkime** [comment]
@jmhammons i haven't see the centering problem you've described. could you provide a screen shot?

as for views that are larger than the screen - the code may need to be improved.

**Comment by jmhammons** [comment]
Yeah, this is what I was seeing...

<img width="404" alt="spinner-centering" src="https://cloud.githubusercontent.com/assets/5428092/13710283/d20b9da4-e775-11e5-85e5-118d514da1aa.png">

**Comment by akdetrick** [comment]
@jmhammons this is now fixed in `v1.5.2`
https://github.com/meetup/sassquatch2/releases/tag/v1.5.2
